src/main/java/com/maplexpbar/MapleXPBarConfig.java

```diff
@@ -32,14 +32,30 @@ default Skill skill()
 
 	@ConfigItem(
 			position = 2,
+			keyName = "showMaxedSkills",
+			name = "Show Maxed Skills",
+			description = "Display bar even if the skill XP exceeds threshold"
+	)
+	default boolean showMaxedSkills() { return false; }
```

Comment:
Defaults to off, not sure what's best here. This fits my personal preference anyhow